classes/submission/reviewer/ReviewerSubmissionDAO.inc.php

```diff
@@ -47,7 +47,33 @@ function getReviewerSubmission($reviewId) {
 		$locale = AppLocale::getLocale();
 		$result = $this->retrieve(
 			'SELECT	a.*,
-				r.*,
+				r.review_id,
+				r.submission_id,
+				r.reviewer_id,
+				r.competing_interests,
+				r.recommendation,
+				r.date_assigned,
+				r.date_notified,
+				r.date_confirmed,
+				r.date_completed,
+				r.date_acknowledged,
+				r.date_due,
+				r.date_response_due,
+				r.last_modified as last_modified_ra,
+				r.reminder_was_automatic,
+				r.declined,
+				r.cancelled,
+				r.reviewer_file_id,
+				r.date_rated,
+				r.date_reminded,
+				r.quality,
+				r.review_round_id,
+				r.stage_id as stage_id_ra,
+				r.review_method,
+				r.round,
+				r.step,
+				r.review_form_id,
+				r.unconsidered,
```

Comment:
This looks good, but I wonder if all of these are necessary. Since this is a light-weight fix for `stable-3_3_0`, can we only specify the columns with `as xxxx`? Somehing like:

```
a.*,
r.*,
r.last_modified as last_modified_ra,
r.stage_id as stage_id_ra,
```

I'm just thinking this would limit the chance that a column gets forgotten.

Reply:
Here I think I have to do it that way. The problem is with the same name columns: for example there is a column named  `stage_id` in both `submissions` and `review_assignments` tables. if I do it like so 

```
a.*,
r.*,
r.last_modified as last_modified_ra,
r.stage_id as stage_id_ra,
```
then the `select` query will return 2 `stage_id` results (the one from the `a.*` and the one from the `r.*`) (along with the named `stage_id_ra`). 

After that, in the `_fromRow` function, when the code tries that `$reviewerSubmission->setStageId($row['stage_id']);` it would pick the second one I suppose (from the `r.*`) when the code needs the first one (from the `a.*`).

That bug was present in the existing code both `stable-3_3_0` and `main` branches. I think that this could be resolved if we changed the select `query` to first take the `r.*` and then the `a.*` but I thought that getting the columns by name for the `review_assignments` table is more clear. 

If you think that this complicates things more than the other solution, I can change it and add a comment regarding the order of the select columns.

Reply:
Makes sense. Thanks for explaining it. Let's leave it the way you've done it. :+1:

Reply:
@asmecher @NateWr I added the following comment on top of the select statement

```
// Each column of review_assignments table is taken separately intentionaly so that
// same named columns like stage_id from different tables is treated separately in _fromRow function
```